### DISCUSSION TITLE
Libero Soc V2021.2 is not available on Microchip site. How can we use these files ?

### DISCUSSION THREAD

**Discussion by tanzilli** [polarfire-soc/polarfire-soc-documentation#81] [Reference designs, FPGA and development kits] [answered]
I'm starting on these days to use the ICICLE Kit.  In the latest README.md file is written "This repository supports Libero SoC V2021.2, which is available for download here". Unfortunately the V2021.2  version is not available on the Microchip web side and the MPFS_ICICLE_KIT_REFERENCE_DESIGN.tcl file works just on V2021.2 version.

The Libero version available are V2021.3 and V2021.1. How can I use this files ?

This is the error during the execute>

Error: Failure when executing Tcl script. [ Line 11: Incorrect Libero version detected. Please use Libero v2021.2 to run these scripts. ]
Error: The Execute Script command failed.

**Comment by eppidei** [comment, marked as answer]
Ciao Sergio,

please try this link
https://www.microchip.com/en-us/products/fpgas-and-plds/fpga-and-soc-design-tools/fpga/libero-software-later-versions

at the bottom of the page you will find the "Documents and Downloads" section and click on Software Downloads.

Hope it helps.

**Comment by tanzilli** [comment]
Hi Eppidei, thanks for your fast reply. I found Libero SoC V2021.2 and it works. Thanks

**Comment by tanzilli** [comment]
Hi Eppidei,

Many thanks for you fast reply. I solves with your link.

Thanks

Sergio

Il giorno mer 26 gen 2022 alle 22:55 eppidei ***@***.***> ha
scritto:

> Ciao Sergio,
>
> please try this link
>
> https://www.microchip.com/en-us/products/fpgas-and-plds/fpga-and-soc-design-tools/fpga/libero-software-later-versions
>
> at the bottom of the page you will find the "Documents and Downloads"
> section and click on Software Downloads.
>
> Hope it helps.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/polarfire-soc/polarfire-soc-documentation/issues/80>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADJ7QEKWO4EBYTEWP6GYN3UYBUTHANCNFSM5M3YIHNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Inviato da Gmail Mobile
